packages/payment-integration-test-framework/src/fixture/pageObject/Checkout.ts

```diff
@@ -16,7 +16,7 @@ export class Checkout {
     }
 
     // CheckoutFixtures helper, only used in ../CheckoutFixtures.ts
-    async setHarFolderPath(file:string): void {
+    async setHarFolderPath(file:string): Promise<void>  {
```

Comment:
This is my mistake. This func is not `async`.
I have removed `async` in today's PR.